### PR TITLE
chore: include api.json into the Playwright package

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -14,30 +14,53 @@
...
-  await gracefullyCloseAll();
-  process.exit(0);
-};
-transport.onmessage = message => dispatcherConnection.dispatch(JSON.parse(message));
-dispatcherConnection.onmessage = message => transport.send(JSON.stringify(message));
+const readFileAsync = util.promisify(fs.readFile);
+const writeFileAsync = util.promisify(fs.writeFile);
+
+export async function copyPrintDeps(destination: string) {
+  const checkDbPath = require.resolve('../bin/PrintDeps.exe');
```
Comment (yury-s) [inline src/server.ts]: typo: checkDepsPath

**Review by yury-s — APPROVED** [review]

File: utils/doclint/generateApiJson.js [hunk 2]
```diff
@@ -59,19 +58,28 @@ function serializeMember(member) {
...
+  delete result.argsArray;
+  delete result.templates;
+  if (result.properties && !Object.keys(result.properties).length)
+    delete result.properties;
+}
+
 function serializeType(type) {
   const result = { ...type };
-  if (type.properties) {
+  if (type.properties && type.properties.length) {
```
Comment (yury-s) [inline utils/doclint/generateApiJson.js:77]: we can also save some bytes by removing other empty/default value fields (required, returnComment etc)

**Review by yury-s — APPROVED** [review]

File: utils/doclint/generateApiJson.js (hunk 2 above)
Comment (pavelfeldman) [reply utils/doclint/generateApiJson.js:77]: It does not save much...

File: src/server.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/server.ts]: Done

**Pull request merged**